File: 2018.1/2018-03-12/Activities/app/src/main/java/br/ufpe/cin/if1001/activities/TxtActivity.java
package br.ufpe.cin.if1001.activities;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class TxtActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_txt);

        Intent i = getIntent();
        String assunto = i.getStringExtra(Intent.EXTRA_SUBJECT);
        String texto = i.getStringExtra(Intent.EXTRA_TEXT);

        TextView t1 = findViewById(R.id.assunto);
        t1.setText(assunto);
        TextView t2 = findViewById(R.id.texto);
        t2.setText(texto);
    }
}

File: 2019-03-20/Activities/app/src/main/java/br/ufpe/cin/if1001/activities/TxtActivity.kt
package br.ufpe.cin.if1001.activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent
import kotlinx.android.synthetic.main.activity_txt.*;

class TxtActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_txt)

        //val i = intent
        assunto.text = intent.getStringExtra(Intent.EXTRA_SUBJECT)
        texto.text = intent.getStringExtra(Intent.EXTRA_TEXT)
    }
}
